pkg/controller/serviceaccount/namespaces.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2020-2024 Project Capsule Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package serviceaccount
+
+import (
+	"context"
+
+	capsulev1beta2 "github.com/projectcapsule/capsule/api/v1beta2"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+// Set the Tenant owner reference on the Namespace specified.
+func (r *ServiceAccountReconciler) setNamespaceOwnerRef(ctx context.Context, ns *corev1.Namespace, tnt *capsulev1beta2.Tenant) error {
+	scheme := runtime.NewScheme()
```

Comment:
You can extract the Scheme from the controller runtime client already injected, if these ones are already registered in the manager setup.

Reply:
Thank you! TIL

Reply:
Applied @prometherion PTAL